modules/db.js

```diff
@@ -92,14 +97,21 @@ module.exports = {
 		ALTER TABLE members
 			ADD COLUMN IF NOT EXISTS group_pos INTEGER;
 
+		SELECT create_constraint_if_not_exists('members','automatic_member_id_fkey',
+			'ALTER TABLE automatic ADD CONSTRAINT automatic_member_id_fkey FOREIGN KEY (member_id) REFERENCES members(id);'
+		);
+		SELECT create_constraint_if_not_exists('automatic','automatic_user_id_key',
+			'ALTER TABLE automatic ADD CONSTRAINT automatic_user_id_key UNIQUE (user_id);'
+		);
 		SELECT create_constraint_if_not_exists('groups','groups_user_id_name_key',
 			'ALTER TABLE groups ADD CONSTRAINT groups_user_id_name_key UNIQUE (user_id, name);'
 		);
 		SELECT create_constraint_if_not_exists('groups','members_group_id_fkey',
 			'ALTER TABLE members ADD CONSTRAINT members_group_id_fkey FOREIGN KEY (group_id) REFERENCES groups(id);'
 		);`);
 
-		await pool.query('CREATE INDEX CONCURRENTLY IF NOT EXISTS members_lower_idx ON members(lower(name))');
+		await pool.query('CREATE INDEX CONCURRENTLY IF NOT EXISTS automatic_user_id ON automatic(user_id);');
```

Comment:
Why?

Reply:
That line that was removed wasn't removed, it should still be there. it may be one line lower or higher than it was?

The index is there because I assume 90% of the querying on the auto table will be on that user_id.  The index in theory will  prevent it from getting laggy as the table grows, but I may be mistaken.